lib/helpers.dart

```diff
@@ -53,7 +53,7 @@ Element? querySelector(String selectors) => document.querySelector(selectors);
 
 bool isInstanceOfDomType(JSObject? o, String domType) {
   if (o == null) return false;
-  final constructor = globalJSObject[domType.toJS];
+  final constructor = globalContext.getProperty(domType.toJS);
```

Comment:
I believe you can do `globalContext[domType]`  (now `[]` takes Strings instead of JSStrings to make it easier to write).